build.cmd

```diff
@@ -209,6 +211,7 @@ if /i "%1" == "-nopgooptimize"       (set __PgoOptimize=0&set processedArgs=!pro
 if /i "%1" == "-ibcinstrument"       (set __IbcTuning=/Tuning&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
 if /i "%1" == "-toolset_dir"         (set __ToolsetDir=%2&set __PassThroughArgs=%__PassThroughArgs% %2&set processedArgs=!processedArgs! %1 %2&shift&shift&goto Arg_Loop)
 if /i "%1" == "-crossgenaltjit"      (set __CrossgenAltJit=%2&set processedArgs=!processedArgs! %1 %2&shift&shift&goto Arg_Loop)
+if /i "%1" == "-amd64hosttools"      (set __Use64BitHostTools=64&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
 
```

Comment:
Should we use `x64` consistently here (instead of the occasional `amd64`)?

Or perhaps the option should simply be `-use64BitHostTools`?

Reply:
Cmake will test to make sure the compiler works and exists. Will make the name change.